### ISSUE TITLE
LESS

### ISSUE THREAD

**Issue by austinstudio** [emmetio/livestyle-sublime-old#110]
How is support for LESS coming?  I tried using this plugin and after taking a while to get it to connect and work, I started to realize that it breaks LESS pretty badly.  As I edit styles that contain variables it converts the variables over to their hard-coded values.  As I edit the style in the LESS file it puts the exact variable text into the browser.  This means that it is completely useless with a pre-processor.

I would love to use this plugin but I just can't right now.  Hope to see updates soon.  Please let me know if the support for LESS has been improved.
##

**Comment by Rad1calDreamer** [comment]
and SASS\SCSS please =)

**Comment by austinstudio** [comment]
Any update on this?  Any chance we could get some sort of support for pre-processors?

**Comment by sergeche** [comment]
Yes, working hard on preprocessor support, stay tuned for updates :)

**Comment by thecotne** [comment]
@sergeche please make on way update 
so i can live code from less

**Comment by Rad1calDreamer** [comment]
we're really excited
